<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Board">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>521</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_RollDice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll Dice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_BuyProperty">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy Property</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_EndTurn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>End Turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_InitGame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Init Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
